share/functions/fish_default_mode_prompt.fish

```diff
@@ -18,6 +18,20 @@ function fish_default_mode_prompt --description "Display vi prompt mode"
             case visual
                 set_color --bold magenta
                 echo '[V]'
+            case operator
```

Comment:
please update the other two occurrences of this switch-case statement as well,
(including the one in docs).

No need to make them as fully-featured as our default;
we could make those two output `[N]` in both normal and operator mode.